helm_deploy/crime-portal-gateway/templates/_envs.tpl

```diff
@@ -34,7 +34,7 @@ env:
         name: crime-portal-gateway-queue-credentials
         key: sqs_id
 
-  - name: AWS_SNS_COURT_CASE_EVENTS_TOPIC
+  - name: HMPPS_SQS_TOPICS_ COURTCASEEVENTSTOPIC_ARN
```

Comment:
the naming is a bit confusing but that's an issue for another day.

**more like:**
HMPPS_SQS ... SNS_TOPIC...ARN

Reply:
missing space is a larger concern!
Agreed and it has caught me out before, but it would be a change to the library